paparazzi-annotations/build.gradle

```diff
@@ -3,5 +3,6 @@ apply plugin: 'org.jetbrains.kotlin.plugin.compose'
 apply plugin: 'com.vanniktech.maven.publish'
 
 dependencies {
-  implementation libs.compose.runtime
+  compileOnly libs.compose.runtime
+  compileOnly libs.tools.layoutlib
```

Comment:
nit: I don't love having to depend on this but we need access to `android.content.res.Configuration` used in preview. Maybe we could hardcode these values instead?